src/validation/tests/snapshots/rusty__validation__tests__variable_length_array_test__assignment__input_by_ref_should_deliver_improvment_suggestion.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: src/validation/tests/variable_length_array_test.rs
+expression: res
+---
+ImprovementSuggestion { message: "Use VAR_IN_OUT instead of VAR_INPUT {ref} if you're assigning to it", range: [SourceRange { range: 353..361 }] }
```

Comment:
I think we can improve this message by telling the user that assigning to VAR_INPUT{ref} will mutate the referenced value, just like VAR_IN_OUT would. so maybe something like "VAR_INPUT {ref} variables are not immutable. Assigning to such a variable will also change the referenced value."
Is there a reason to use VAR_IN_OUT instead, other than perceived/expected mutability?

Reply:
> I think we can improve this message by telling [...]

Yup, thanks. I intended to change the message but totally forgot about it. 

> Is there a reason to use VAR_IN_OUT instead, other than perceived/expected mutability?

I assume because this is meant to be a use-case of IN_OUT rather than INPUT {ref}